datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -60,8 +60,6 @@ protobuf==3.20.2; python_version > '3.0'
 psutil==5.9.0
 psycopg-pool==3.1.8; python_version > '3.0'
 psycopg2-binary==2.9.9; python_version > '3.0'
-psycopg[binary]==3.1.12; sys_platform == 'win32' and python_version > '3.0'
-psycopg[c]==3.1.12; sys_platform != 'win32' and python_version > '3.0'
```

Comment:
nit: I'd do this in a separate PR that targets master

Reply:
Agreed this should come separately first

Reply:
OK, removed.

Reply:
https://github.com/DataDog/integrations-core/pull/16538